front/app/components/SeatInfo/index.tsx

```diff
@@ -52,6 +60,9 @@ const SeatInfo = ({ seatType }: Props) => {
     collaborator: seats.data.attributes.project_moderators_number,
   }[seatType];
   const additionalSeats = currentSeatNumber - maximumSeatNumber;
+  const showAdditionalSeats = Boolean(
+    additionalSeats > 0 && maximumAdditionalSeats
+  );
```

Comment:
The Boolean kind of hides the types (or makes it harder to deduct what we want to know). Maybe we should be more explicit and use 

`additionalSeats > 0 && !isNil(maximumAdditionalSeats)
`
or

`additionalSeats > 0 && typeof maximumAdditionalSeats === 'number'
`

Reply:
🤔 Theoretically, the above cases don't cover zero as an option. Which while unexpected to happen, is a good idea for the frontend to defensively not show.

It would then have to be `additionalSeats > 0 && typeof maximumAdditionalSeats === 'number' && maximumAdditionalSeats > 0` which I think is harder to understand than the Boolean. What other meaning might the boolean have / confused with?

Reply:
I mean that we're saying "convert this value to a boolean", which is a cover-up for being more specific about what we want. That's all. Feel free to handle it as you see fit.